common/utils/src/main/resources/error/error-conditions.json

```
@@ -3923,6 +3923,13 @@
     ],
     "sqlState" : "0A000"
   },
+  "NEGATIVE_SCALE_DISALLOWED": {
+    "message": [
+      "Negative scale is not allowed: '<scale>'.",
+      "Set the config <sqlConf> to 'true' to allow it."
```

Let's follow the existing convention SQL conf values in `error-conditions.json`:
```suggestion
      "Negative scale is not allowed: '<scale>'. Set the config <sqlConf> to \"true\" to allow it."
```